README.md

```diff
@@ -20,6 +20,7 @@ before you build.
 Once you've set up your development environment, let's build the plugin.
 
 ```sh
+$ ./hack/update-deps.sh
 $ go build -o kn-event ./cmd/kn-event/main.go
```

Comment:
We should use mage to build & test this plugin

```suggestion
$ mage build
```

Reply:
Is `mage` available everywhere ootb ? If not, the installation of mage should be added to the script (i.e. checking if it exists and if not `go get` it)

Reply:
There's a no installation way. By using small script provided. Just call:

`./mage build`